specification/maps/resource-manager/Microsoft.Maps/preview/2020-04-01-preview/maps-management.json

```diff
@@ -0,0 +1,1143 @@
+          },
+          {
+            "name": "PrivateAtlasCreateParameters",
+            "in": "body",
+            "description": "The new or updated parameters for the Private Atlas resource.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlasCreateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Private Atlas was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "201": {
+            "description": "The Private Atlas was successfully created.",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "202": {
+            "description": "The Private Atlas was accepted for creation",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "description": "Updates the Private Atlas resource. Only a subset of the parameters may be updated after creation, such as Tags.",
+        "operationId": "PrivateAtlasCollection_Update",
+        "x-ms-examples": {
+          "UpdateAccount": {
+            "$ref": "./examples/UpdatePrivateAtlas.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateAtlasNameParameter"
+          },
+          {
+            "name": "PrivateAtlasUpdateParameters",
+            "in": "body",
+            "description": "The updated parameters for the Private Atlas.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlasUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Private Atlas was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "202": {
+            "description": "The Private Atlas was accepted for update",
+            "schema": {
+              "$ref": "#/definitions/PrivateAtlas"
+            }
+          },
+          "404": {
+            "description": "The specified Private Atlas was not found. Nothing was updated."
+          },
```

Comment:
this will fall under the "default" error path, you shouldn't manually specify normal errors